install_config/registry/deploy_registry_existing_clusters.adoc

```diff
@@ -91,7 +94,147 @@ Daemonsets ensure that when nodes are created, they contain copies of a
 specified pod. When the nodes are removed, the pods are garbage collected.
 
 For more information on `DaemonSets`, see
-xref:../../dev_guide/daemonsets.adoc#dev-guide-daemonsets[Using Daemonsets].
+xref:../../dev_guide/daemonsets.adoc#dev-guide-daemonsets[Using DaemonSets].
+
+[[redeploy-registry]]
+=== Re-deploying the Registry
+
+There is a couple of things to keep on mind when re-deploying an existing docker-registry
+DeploymentConfig or DaemonSet:
+
+1. Service IP shall be preserved.
+2. Any change to the service IP is not propagated to the {product-title} cluster's components
+until the master API and controllers are restarted.
+3. The docker-registry service must exist before the docker-registry pod is created.
+
+**Service IP shall be preserved** because master caches the docker-registry's internal address.
```

Comment:
again unclear if this means "you must keep the service ip the same" or "the system will keep the service ip the same"